Old gateway UpgradeTest fail

**Description**
After activating the UpgradeTest again we saw that the tests with the old standalone gateway fail.
It seems that the Gateway is not able to contact with the broker.

Test fails with:
```
2020-04-16 05:47:19.177 [io.zeebe.gateway.impl.broker.BrokerRequestManager] [gateway-scheduler-zb-actors-0] ERROR io.zeebe.gateway - Error handling gRPC request
io.grpc.StatusRuntimeException: DEADLINE_EXCEEDED: Time out between gateway and broker: Request timed out after PT15S
	at io.grpc.Status.asRuntimeException(Status.java:524) ~[grpc-api-1.28.1.jar:1.28.1]
	at io.zeebe.gateway.EndpointManager.convertThrowable(EndpointManager.java:397) ~[zeebe-gateway-0.23.0.jar:0.23.0]
	at io.zeebe.gateway.EndpointManager.lambda$sendRequest$3(EndpointManager.java:311) ~[zeebe-gateway-0.23.0.jar:0.23.0]
	at io.zeebe.gateway.impl.broker.BrokerRequestManager.lambda$sendRequest$3(BrokerRequestManager.java:148) ~[zeebe-gateway-0.23.0.jar:0.23.0]
	at io.zeebe.gateway.impl.broker.BrokerRequestManager.lambda$sendRequestInternal$5(BrokerRequestManager.java:191) ~[zeebe-gateway-0.23.0.jar:0.23.0]
	at io.zeebe.util.sched.future.FutureContinuationRunnable.run(FutureContinuationRunnable.java:33) [zeebe-util-0.23.0.jar:0.23.0]
	at io.zeebe.util.sched.ActorJob.invoke(ActorJob.java:76) [zeebe-util-0.23.0.jar:0.23.0]
	at io.zeebe.util.sched.ActorJob.execute(ActorJob.java:39) [zeebe-util-0.23.0.jar:0.23.0]
	at io.zeebe.util.sched.ActorTask.execute(ActorTask.java:115) [zeebe-util-0.23.0.jar:0.23.0]
	at io.zeebe.util.sched.ActorThread.executeCurrentTask(ActorThread.java:107) [zeebe-util-0.23.0.jar:0.23.0]
	at io.zeebe.util.sched.ActorThread.doWork(ActorThread.java:91) [zeebe-util-0.23.0.jar:0.23.0]
	at io.zeebe.util.sched.ActorThread.run(ActorThread.java:195) [zeebe-util-0.23.0.jar:0.23.0]
Caused by: java.util.concurrent.TimeoutException: Request timed out after PT15S
	at io.zeebe.transport.impl.RequestContext.timeout(RequestContext.java:109) ~[zeebe-transport-0.23.0.jar:0.23.0]
	at io.zeebe.transport.impl.AtomixClientTransportAdapter.timeoutFuture(AtomixClientTransportAdapter.java:131) ~[zeebe-transport-0.23.0.jar:0.23.0]
	at io.zeebe.transport.impl.AtomixClientTransportAdapter.lambda$sendRequestWithRetry$0(AtomixClientTransportAdapter.java:60) ~[zeebe-transport-0.23.0.jar:0.23.0]
	... 6 more


io.zeebe.client.api.command.ClientStatusException: Time out between gateway and broker: Request timed out after PT15S

	at io.zeebe.client.impl.ZeebeClientFutureImpl.transformExecutionException(ZeebeClientFutureImpl.java:93)
	at io.zeebe.client.impl.ZeebeClientFutureImpl.join(ZeebeClientFutureImpl.java:50)
	at io.zeebe.test.UpgradeTestCase$TestCaseBuilder.lambda$deployWorkflow$4(UpgradeTestCase.java:61)
	at java.base/java.util.function.Consumer.lambda$andThen$0(Consumer.java:65)
	at io.zeebe.test.UpgradeTestCase.setUp(UpgradeTestCase.java:29)
	at io.zeebe.test.UpgradeTest.oldGatewayWithNewBroker(UpgradeTest.java:180)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.util.concurrent.ExecutionException: io.grpc.StatusRuntimeException: DEADLINE_EXCEEDED: Time out between gateway and broker: Request timed out after PT15S
	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:395)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1999)
	at io.zeebe.client.impl.ZeebeClientFutureImpl.join(ZeebeClientFutureImpl.java:48)
	... 18 more
Caused by: io.grpc.StatusRuntimeException: DEADLINE_EXCEEDED: Time out between gateway and broker: Request timed out after PT15S
	at io.grpc.Status.asRuntimeException(Status.java:533)
	at io.grpc.stub.ClientCalls$StreamObserverToCallListenerAdapter.onClose(ClientCalls.java:449)
	at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:426)
	at io.grpc.internal.ClientCallImpl.access$500(ClientCallImpl.java:66)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.close(ClientCallImpl.java:689)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl.access$900(ClientCallImpl.java:577)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:751)
	at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:740)
	at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	... 1 more
```

Zbctl shows that the gateway doesnt know any brokers.
```
[zell dist/ ns:default]$ ./zbctl --insecure --address=0.0.0.0:32791 status
Cluster size: 0
Partitions count: 0
Replication factor: 0
Gateway version: 0.23.0
Brokers:
```

I currently assume that the env variable which is set inside the container is wrong.
If we we set the contact point at the Gateway builder it will set as environment variable the following:

```
ZEEBE_GATEWAY_CONTACT_POINT -> localhost:32811 
```

The config is printed after starting the gateway:

```
2020-04-16 05:46:59.130 [] [main] INFO  io.zeebe.gateway - Version: 0.23.0
2020-04-16 05:46:59.201 [] [main] INFO  io.zeebe.gateway - Starting standalone gateway with configuration {
  "network" : {
    "host" : "0.0.0.0",
    "port" : 26500,
    "minKeepAliveInterval" : "PT30S"
  },
  "cluster" : {
    "contactPoint" : "127.0.0.1:26502",
    "requestTimeout" : "PT15S",
    "clusterName" : "zeebe-cluster",
    "memberId" : "zeebe-gateway-0",
    "host" : "zeebe-gateway-zjDiXB",
    "port" : 26502
  },
  "threads" : {
    "managementThreads" : 1
  },
  "monitoring" : {
    "enabled" : false,
    "host" : "0.0.0.0",
    "port" : 9600
  },
  "security" : {
    "enabled" : false,
    "certificateChainPath" : null,
    "privateKeyPath" : null
  }
}
```

We see no change when we set a different contact point.
Taking a look at the Gateway config it seems that the contact point is actually set at `GatewayCfg.ClusterCfg.ContactPoint`. In the gateway default config file we see that the environment variable is names as `ZEEBE_GATEWAY_CLUSTER_CONTACTPOINT`.

I think this is the reason why the test fails. I will ignore the test for now and we should activate him after we fixed this issue.

--- a/upgrade-tests/src/test/java/io/zeebe/test/ContainerStateRule.java
+++ b/upgrade-tests/src/test/java/io/zeebe/test/ContainerStateRule.java
@@ -41,7 +41,7 @@ class ContainerStateRule extends TestWatcher {
   }
 
   @Override
-  protected void failed(Throwable e, Description description) {
+  protected void failed(final Throwable e, final Description description) {
     super.failed(e, description);
     if (broker != null) {
       log("Broker", broker.getLogs());
@@ -53,7 +53,7 @@ class ContainerStateRule extends TestWatcher {
   }
 
   @Override
-  protected void finished(Description description) {
+  protected void finished(final Description description) {
     close();
   }
 
@@ -73,6 +73,7 @@ class ContainerStateRule extends TestWatcher {
     broker =
         new ZeebeBrokerContainer(brokerVersion)
             .withFileSystemBind(volumePath, "/usr/local/zeebe/data")
+            .withEnv("ZEEBE_BROKER_CLUSTER_CLUSTERNAME", "zeebe-cluster")
             .withNetwork(network)
             .withEmbeddedGateway(gatewayVersion == null)
             .withLogLevel(Level.DEBUG);
@@ -83,7 +84,8 @@ class ContainerStateRule extends TestWatcher {
     if (gatewayVersion != null) {
       gateway =
           new ZeebeStandaloneGatewayContainer(gatewayVersion)
-              .withContactPoint(broker.getContactPoint())
+              .withEnv("ZEEBE_GATEWAY_CLUSTER_CONTACTPOINT", broker.getContactPoint())
+              .withEnv("ZEEBE_GATEWAY_CLUSTER_CLUSTERNAME", "zeebe-cluster")
               .withNetwork(network)
               .withLogLevel(Level.DEBUG);
       gateway.start();

--- a/upgrade-tests/src/test/java/io/zeebe/test/UpgradeTest.java
+++ b/upgrade-tests/src/test/java/io/zeebe/test/UpgradeTest.java
@@ -27,7 +27,6 @@ import java.util.Optional;
 import java.util.concurrent.TimeUnit;
 import org.agrona.IoUtil;
 import org.assertj.core.util.Files;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.RuleChain;
@@ -171,7 +170,6 @@ public class UpgradeTest {
   }
 
   @Test
-  @Ignore
   public void oldGatewayWithNewBroker() {
     // given
     state

